js/privatebin.js

```diff
@@ -4280,7 +4519,7 @@ jQuery.PrivateBin = (function($, RawDeflate) {
          */
         me.isAttachmentReadonly = function()
         {
-            return createButtonsDisplayed && $attach.hasClass('hidden');
+            return !createButtonsDisplayed || $attach.hasClass('hidden');
```

Comment:
I cannot reproduce the error after applying my bugfixes, I think I changed a lot of things, even hiding the attachment file select input to make everything work as expected.

I think there are a lot of edge cases making this part of logic more than tedious, for exmaple clone from paste reading view vs. clone from paste successfully created view.

I'll try to deploy from current master branch and see what happens.

I would suggest we accept this part of bugfix as a whole, including but not limited to:
1. ghost attachment attached to paste even though not displayed when you create a new paste by clicking new button on a paste with attahment, seemed loaded file not cleaned up properly.
2. drag and drop filename conflict with click and select dialog.
3. attachment dropdown incorrectly suggest file select input and drag & drop can be used together
4. and so on...

I'll setup a demo site very soon so you can test how it works after my changes are applied.

Reply:
OK I misread your message, I'm able to reproduce this bug even after applying my bugfixes, I get what is going on now, I'll fix this problem immediately.

Reply:
Proper way of handling this would be let go of any paste item that is not file type:

```
        /**
         * attaches the clipboard attachment handler to the page
         *
         * @name   AttachmentViewer.addClipboardEventHandler
         * @private
         * @function
         */
        function addClipboardEventHandler() {
            $(document).on('paste', function (event) {
                const items = (event.clipboardData || event.originalEvent.clipboardData).items;
                const lastItem = items[items.length - 1];
                if (lastItem.kind === 'file') {
                        if (TopNav.isAttachmentReadonly()) {
                            event.stopPropagation();
                            event.preventDefault();
                            return false;
                        } else {
                            readFileData(lastItem.getAsFile());
                        }
                }
            });
        }
```

Reply:
Fixed and latest demo should allow pasting into password modal meanwhile dropzone & paste from clipboard working whenever modification of attachment is possible.